demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2504,7 +2521,21 @@ export class DemoMeetingApp
   async authenticate(): Promise<string> {
     const joinInfo = (await this.joinMeeting()).JoinInfo;
     const configuration = new MeetingSessionConfiguration(joinInfo.Meeting, joinInfo.Attendee);
-    await this.initializeMeetingSession(configuration);
+    const { Meeting: { MeetingId, MediaPlacement: { EventIngestionUrl } }} = joinInfo.Meeting;
```

Comment:
nit: Can this be const { MeetingId, MediaPlacement: { EventIngestionUrl } } = jobInfo.Meeting.Meeting like below?

Reply:
Since the URL is now from meeting session configuration, this has been removed.